controlpanel/api/tools.py

```diff
@@ -111,9 +111,9 @@ def deploy_for(self, user):
             *params,
         )
 
-    def get_user_deployment(self, user):
-        load_kube_config()
-        deployments = kubernetes.client.AppsV1Api().list_namespaced_deployment(
+    def get_user_deployment(self, request):
+        client = RequestUserKubernetesClient()
```

Comment:
:+1:

Reply:
ugh, need to change `request` arg back to `user`